dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -144,6 +144,7 @@
 				CacheDirectory=$(_LinkerCacheDirectory)
 				Debug=$(_BundlerDebug)
 				DeploymentTarget=$(_MinimumOSVersion)
+				@(_XamarinFrameworkAssemblies -> 'FrameworkAssembly=%(Filename)')
```

Comment:
> @(_XamarinFrameworkAssemblies -> 'FrameworkAssembly=%(Filename)')

I think it is the first time I see this `->` what does it do? that did not google very well. I'll give it just a guess, does it give you a chance to shape the data your way when container `@()` is resolved? that is my guess

Reply:
It's called an MSBuild Item Transform: https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-transforms?view=vs-2019

And yes, it's the MSBuild version of `Array.ConvertAll (_XamarinFrameworkAssemblies, (v) => $"FrameworkAssembly={v}")`

Reply:
Thanks 👍 I've only seen it to call functions like IndexOf and friends but not in this context.